config/locales/en.yml

```diff
@@ -654,6 +655,10 @@ en:
   instructions_after_logout: "You can sign in again by clicking %{signin}."
   instructions_after_error: "You can try to sign in again by clicking %{signin}. If the error persists, ask your admin for help."
 
+  label_access_token: "Access token"
+  label_access_token_action_header: "Action"
+  label_access_token_expiration_header: "Expires"
+  label_access_token_expiration_indefinite: "Never"
```

Comment:
You should probably rather use nested attributes here (something we are doing too seldom in general).

Idea:

````yaml
my_account:
  access_token:
    action_header: "Foo"
    expiration_foo: "Bar"
````